server/db/migrations/20200918103933_users.js

```diff
@@ -7,7 +7,6 @@ exports.up = (knex) => {
     table.string('username')
     table.string('email')
     table.string('hash')
-    table.boolean('is_admin')
```

Comment:
This column should be removed in `#####_usersAuth0.js`. It was there in the initial auth0 change, but I was naughty and changed the migrations.